src/styles/utils/colorManipulator.tsx

```diff
@@ -0,0 +1,17 @@
+import { hexToRgb } from '@material-ui/core';
```

Comment:
Should this file still be included? Looks like you changed it `.ts` and made changes above